common/network-common/src/main/java/org/apache/spark/network/server/OneForOneStreamManager.java

```diff
@@ -43,6 +43,7 @@
 
   private final AtomicLong nextStreamId;
   private final ConcurrentHashMap<Long, StreamState> streams;
+  private final AtomicLong totalChunksBeingTransferred = new AtomicLong(0);
```

Comment:
nit: maybe rename to `numChunksBeingTransferred`? Because it's not accumulating all the chunks that are transferred in history.

Reply:
> nit: maybe rename to `numChunksBeingTransferred`? Because it's not accumulating all the chunks that are transferred in history.

Updated